community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v2_3/GeneratedQueryStructure.scala

```diff
@@ -714,6 +714,13 @@ private case class Method(fields: Fields, generator: CodeBlock, aux:AuxGenerator
     }
   }
 
+  override def indexUniqueSeek(nodeVar: String, descriptorVar: String, value: Expression) = {
```

Comment:
Again with the naming. I think we should try to be consistent and call this one `uniqueIndexSeek`.